app/controllers/api_controller/logger.rb

```diff
@@ -1,5 +1,40 @@
 class ApiController
   module Logger
+    def log_request_initiated
+      @requested_at = Time.now.utc
+      api_log_info(" ")
```

Comment:
presumably this is to have `api_log_info` put something on the log without a specific message? Could that method be changed to do that without having to pass it something like `" "`?